skyrim-platform/tools/plugin-example/README.md

```diff
@@ -2,22 +2,27 @@
 
 ## Setting Up
 
-1. Run `npm i` on the command line to install the dependencies.
-2. Create a file `tsc/config.js` with the following contents:
-   ```js
+
+1. Grab the latest version of the Skyrim Platform from https://www.nexusmods.com/skyrimspecialedition/mods/54909?tab=files
+2. Extract the zip and place the files in your Skyrim install folder (Skyrim Special Edition\Data) 
+   ### DO NOT USE A MOD MANAGER AS HOT CODE RELOADING WILL NOT WORK WITH THE VIRTUAL DRIVE.
+3. Copy the `Skyrim Special Edition\Data\Platform\Modules\SkyrimPlatform.ts` file to the root Skyrim folder Skyrim Special Edition
```

Comment:
@generalconsensus Hey! It seems this step isn't really required. `skyrimPlatform.ts` is already part of the nexus mods archive.